java/messages/src/main/java/com/nedap/retail/messages/Client.java

```diff
@@ -248,7 +248,7 @@ public Long articleQuantity() {
             }
         }
 
-        return get(target, new GenericType<List<Article>>() {
+        return post(target, new GenericType<List<Article>>() {
```

Comment:
Maybe good to add comment in code why we do `post` and not `get` when retrieving data from server.